modules/ROOT/partials/configuration/convert_fonts_to_spans.adoc

```diff
@@ -1,7 +1,7 @@
 [[convert_fonts_to_spans]]
 == convert_fonts_to_spans
 
-include::partial$DEPRECATED/generic-5_10.adoc[]
+IMPORTANT: This option has been deprecated in {productname} 5.10 and may be removed in {productname} 6.0.
```

Comment:
This isn't correct as this is the 6.0 docs... also we should update the original admonition not this. We probably want to say something like this:
```
IMPORTANT: This option has been deprecated in {productname} 5.10 and may be removed in a future major {productname} release.
```

- [ ] Revert and fix original admonition instead